### ISSUE TITLE
Most of the projects listed @ board are open source - secure additional funding via channels like "Gitcoin Grants", "Codefund", "GitHub sponsors"

### ISSUE THREAD

**Issue by ideas-into-software** [zerocracy/zerocracy.github.io#91]
Most of the projects listed @ board are open source. 

 Take advantage of open source funding channels to secure additional funds for those projects. 

 Options which are available:

   * "Gitcoin Grants" (https://gitcoin.co/grants/) - both one time and recurring payments
	"Gitcoin Grants has facilitated the distribution of $219k to Open Source since it's launch in January 2019."

  * "Codefund" (https://codefund.app/) 

  * "GitHub sponsors" (https://help.github.com/en/articles/about-github-sponsors) - launched 2 days ago, now in beta

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@ideas-into-software[/z](https://www.0crat.com/u/ideas-into-software) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by g4s8** [comment]
@ideas-into-software I don't think it's Zerocracy issue - each project owner may decide to publish the project on these resources. Zerocracy just provides a team and manager-bot for projects on board.

**Comment by ideas-into-software** [comment]
> @ideas-into-software I don't think it's Zerocracy issue - each project owner may decide to publish the project on these resources. Zerocracy just provides a team and manager-bot for projects on board.

Hi, 

I joined Zerocracy yesterday (May 24). First thing I noticed, after going through all the steps related to on-boarding is that most projects @ https://www.0crat.com/board are listed as having no funds or very little funds. When I talked to George Aristy regarding joining one of such projects for few tasks he mentioned it is paused due to lack of funds, and for almost a year already.

That is the origin from my side for posting this issue - i.e. https://github.com/zerocracy/zerocracy.github.io/issues/91 - as well as https://github.com/zerocracy/zerocracy.github.io/issues/92. 

It's a great tool you built / way of working / but without funds no one, including myself, will work for free. 

So what I can propose from my side - both for #91 and #92 - I can take care of making this happen, starting with detailed specs / scoping both, with you / the team / involved, and take care of implementation for most of the tasks I foresee for #91 and at least part of #92, to resolve this funding issue as soon as possible.

If you'd like me to start, projects for both would have to be created - they can even be "sandboxed", and I can work at that minimum rate 16 USD to make this happen, but for max of 1,5 - 2 weeks. This should be enough time to have majority of #91 and part of #92 done. 

If you'd like me to proceed, please let me know and we can discuss this further. 

Regards,
Michal

**Comment by g4s8** [comment]
@ideas-into-software I agree that it's an issue, but projects listed on board is not a part of Zerocracy - you can create and publish your own project to this board if you want, so I'd suggest moving this ticket to one of the project you want to contribute or contact project owner of such project directly.
